src/vl-map.js

```diff
@@ -58,12 +58,25 @@ export class VlMap extends vlElement(HTMLElement) {
   /**
    * Geeft de OpenLayers map terug.
    *
-   * @Return {VlCustomMap}
+   * @return {VlCustomMap}
    */
   get map() {
     return this._map;
   }
 
+  /**
+   * Geeft de OpenLayers kaart visualisatie terug.
+   *
+   * @return {Object}
+   */
+  get view() {
+    return this._map.getView();
+  }
+
+  get layers() {
+    return [...this.querySelectorAll('vl-map-layer')];
```

Comment:
is vl-map-layer en de bijhorende getter hier dan niet een beetje te generiek eigenlijk? ik vermoed dat het "alles behalve de baselayers", dus de "overlays" zijn? of is dat niet de correcte term?

Reply:
@Pieter-Beckers to the rescue. Ik denk dat overlays != layers en dacht dat het verschil tussen layers/baselayers bestaat en duidelijk genoeg is.

Reply:
ja t zou kunnen, alleen een beetje raar dan dat een baseLAYER geen layer zou zijn?

Reply:
Offline besproken en conclusie om `getNonBaseLayers` te gebruiken. In kader van dit overleg heb ik issue #178
aangemaakt om een naamswijziging te overwegen.